index.html

```diff
@@ -10,7 +10,7 @@
   <link href="main.css" rel="stylesheet">
   <link rel="preconnect" href="https://fonts.gstatic.com">
   <link href="https://fonts.googleapis.com/css2?family=Inter:wght@200;400&display=swap" rel="stylesheet">
-  <title>Portfolio</title>
+  <title>Simon's Portfolio</title>
```

Comment:
It is nice that you've updated the title of your portfolio :+1: 
However, as mentioned in my review summary, you need to add the link to the corresponding line of code in your Pull Request changes.
You can follow the template below in preparing your PR changes

| Checks | Reports | Lines of Code |
| ---------- | ----------- | ------------------ |
| Page Title | Changed to a descriptive title | https://github.com/SimonGrchevski/Portfolio/blob/284b2ade5d1735db3125ca0fe4b882e1d7001d12/index.html#L13 |

Reply:
Well addressed the issue